recipes/sunraster/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "sunraster" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "a46be4d4cc486004830ab8135fe843ae37577d422c55e89ec05e5b5212261322" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
I think the error you are having might be from having a version restriction under `run` that's missing in `host`